manifest.json

```diff
@@ -1,6 +1,7 @@
 {
     "description": "RAML files listed in order corresponding RAML feature appears in RAML 1.0 spec",
     "filePaths": [
+        "tests/raml-1.0/Root/yaml-12-compliancy/valid.raml",
```

Comment:
Is there some YAML <1.2 syntax we could add as "invalid" test case?

Reply:
Yes. Added.